src/OrchardCore.Modules/OrchardCore.MiniProfiler/Manifest.cs

```diff
@@ -4,7 +4,20 @@
     Name = "Mini Profiler",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
-    Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Adds the Mini Profiler widget to pages.",
-    Category = "Performance"
+    Version = ManifestConstants.OrchardCoreVersion
 )]
+
+[assembly: Feature(
+    Id = "OrchardCore.MiniProfiler",
+    Name = "Mini Profiler",
+    Category = "Performance",
+    Description = "Adds the Mini Profiler widget to pages."
```

Comment:
Adds the Mini Profiler widget to front pages.

Reply:
"front end" pages?

Reply:
Right, but why I saw it on other place as front pages, I didn't introduce "end" ;)

Reply:
The other feature uses that terminology.

Reply:
I mean this module